src/Tapper/TypeMappers/DictionaryTypeMappers.cs

```diff
@@ -25,7 +25,7 @@ public string MapTo(ITypeSymbol typeSymbol, ITranspilationOptions options)
             var mapper0 = options.TypeMapperProvider.GetTypeMapper(namedTypeSymbol.TypeArguments[0]);
             var mapper1 = options.TypeMapperProvider.GetTypeMapper(namedTypeSymbol.TypeArguments[1]);
 
-            return $"{{ [key: {mapper0.MapTo(namedTypeSymbol.TypeArguments[0], options)}]: {mapper1.MapTo(namedTypeSymbol.TypeArguments[1], options)} }}";
+            return $"Record<{mapper0.MapTo(namedTypeSymbol.TypeArguments[0], options)}, {mapper1.MapTo(namedTypeSymbol.TypeArguments[1], options)}>";
```

Comment:
Add `undefined` in case the key does not exist.

```suggestion
            return $"Record<{mapper0.MapTo(namedTypeSymbol.TypeArguments[0], options)}, {mapper1.MapTo(namedTypeSymbol.TypeArguments[1], options)} | undefined>";
```

Reply:
This is something I though about as well, but then it's probably better to use `Partial<Record<Key, Value>>` instead to make the missing keys of missing values optional.
Otherwise you'd have to do this in TypeScript to initialize an object of that type with enum keys:
```
enum MyEnum {
    Value1 = 1,
    Value2 = 2,
    Value3 = 3,
}
type MyType = {
    myProperty: Record<MyEnum, number | undefined>;
}

// issue mentioned above
const obj: MyType = {
    [MyEnum.1] = 12345,
    [MyEnum.2] = undefined,
    [MyEnum.3] = undefined,
}
```

Reply:
Let's adopt `Partial<Record<TKey, TValue>>.`